index.js

```diff
@@ -203,6 +205,7 @@ module.exports = function (config) {
 
   if (koop.config.export_workers) {
     var kue = require('kue')
+
     koop.Exporter.export_q = kue.createQueue({
```

Comment:
also probably not as part of this PR, but it seems weird to handle this export_q stuff here